README.md

```diff
@@ -49,6 +48,60 @@ def example_s3_upload_flow():
 example_s3_upload_flow()
 ```
 
+### Write and run a flow with AwsCredentials and S3Bucket
+
+```python
+import asyncio
+from prefect import flow
+from prefect_aws import AwsCredentials, S3Bucket
```

Comment:
I think the imports needs to be
`from prefect_aws.s3 import S3Bucket`

Reply:
[Update examples in readme](https://github.com/PrefectHQ/prefect-aws/pull/47/commits/a56695de2dee9d2118c6b2a66e42f44cd030a174)